src/entities/communitySbt.ts

```diff
@@ -315,52 +316,104 @@ class SBT {
         seasonStart: number,
         seasonEnd: number,
       }): Promise<BadgeResponse[]> {
-        try {
-            let badgesResponse : BadgeResponse[] = [];
-
-            // programmatic badges
-            if (this.badgesSubgraphUrl) {
-                const badgeQuery = `
-                    query( $id: String) {
-                        seasonUser(id: $id) {
-                            id
-                            badges {
-                            id
-                            awardedTimestamp
-                            mintedTimestamp
-                            badgeType
-                            }
-                        }
-                    }
-                `;
-                const client = new ApolloClient({
-                    cache: new InMemoryCache(),
-                    link: new HttpLink({ uri: this.badgesSubgraphUrl, fetch })
+
+        // past season
+        if (seasonEnd < DateTime.now().toSeconds()) {
+            const badges = await this.getOldSeasonBadges({
+                userId,
+                seasonId,
+                seasonStart,
+                seasonEnd,
+            })
+            return badges;
+        }
+
+        const badges = await this.computeSeasonBadges({
+            userId,
+            seasonId,
+            seasonStart,
+            seasonEnd,
+        });
+        return badges;
+
+    } 
+    
+    public async getOldSeasonBadges({
```

Comment:
is there a unit test for this function?

Reply:
no tests, we'll have to QA this internally